sphinx/themes/basic/static/basic.css_t

```diff
@@ -114,16 +114,25 @@ img {
 
 /* -- search page ----------------------------------------------------------- */
 
-ul.search {
-    margin: 10px 0 0 20px;
-    padding: 0;
-}
-
 ul.search li {
     padding: 5px 0 5px 20px;
-    background-image: url(file.png);
-    background-repeat: no-repeat;
-    background-position: 0 7px;
+    list-style: "\25A1";  /* Unicode: White Square */
+}
+
+ul.search li.title {
+    list-style: "\1F4C1";  /* Unicode: File Folder */
+}
```

Comment:
This is the only annotation symbol that I find slightly confusing / out of context, personally.

It does make some sense to me that it's for use to represent that titles are 'containers' in some sense for text content.

I'm browsing unicode character references to try to find something else that could represent text/documents, headings/titles, and/or the potential for containership/nesting.

Reply:
Maybe it's a controversial misuse, but perhaps the paragraph symbol (called [Pilcrow](https://en.wikipedia.org/wiki/Pilcrow), as I learned today)?

Reply:
(or we could try to bring the [capitulum](https://en.wikipedia.org/wiki/Capitulum) back into style)

Reply:
Another alternative could simply be to style `title` and `text` matches identically to begin with.

I'll try to spend a bit more time thinking about this before adding any further thoughts/comments.

Reply:
The idea behind folder was "structuring element" but I'm not too convinced of it either.

Reply:
On reflection: I'd recommend keeping this simple, and using the same styling for both `title` and `text` search results - that is, `1F4C4` (unicode: page facing).

We could still _emit_ the CSS classes `title` and `text` in the result list, allowing downstream sites to customize their display to suit their projects if they choose.

Some reasoning for this suggestion: with a default Sphinx configuration and HTML build, it is possible to infer from each listed search result whether the input search query term(s) matched in the corresponding document's title and/or content body.  In my personal opinion it is a nice addition to be able to distinguish between glossary items and programming objects, but seems lower priority to distinguish page title matches from other page content matches.